### PR TITLE
Make CircleCI push image to gcr

### PR DESCRIPTION
**Pull request by kshahat**
Create and push image to GCR using CircleCI. This took way too long but I learned a lot about docker and CircleCI

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 NS = default
 
 SERVICE_NAME = bigtable-autoscaler
-SERVICE_IMAGE_VERSION ?= 0.0.1 
+SERVICE_IMAGE_VERSION = latest
```
Comment (kshahat) [inline Makefile:4]: @alexagriffith you were right lol

**Review by shraykay — APPROVED** [review]

**Review by bennorton767 — APPROVED** [review]

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,42 +1,38 @@
...
-      # - image: circleci/postgres:9.4
-
-    working_directory: ~/repo
-
-    environment:
-      # Customize the JVM maximum heap limit
-      MAVEN_OPTS: -Xmx3200m
-
+    machine:
+      image: ubuntu-1604:202004-01
```
Comment (dangermike) [inline .circleci/config.yml:5]: 16.04? It doesn't really matter, but why so back-versioned?

**Review by dangermike — APPROVED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (kshahat) [reply .circleci/config.yml:5]: didn't know where to find newer versions and this was the latest I could find from the CI page 🤷‍♂️

Comment (alexagriffith) [reply .circleci/config.yml:5]: I think you can use any of these https://hub.docker.com/_/ubuntu/ but idk

**Review by alexagriffith — COMMENTED** [review]
lgtm

File: .circleci/config.yml (hunk 2 above)
Comment (kshahat) [reply .circleci/config.yml:5]: cool; good to know

**Pull request merged**